README.rst

```diff
@@ -184,6 +184,15 @@ endpoint, which provides a paginated list of such locations in GeoJSON format:
 
     GET /api/v1/device/geojson/
 
+Subnet Division App
+~~~~~~~~~~~~~~~~~~~
+
+This app enables controller to automatically allocate subnets and IP addresses to
+devices based on existing subnet division rules created by user.
```

Comment:
We can shorten to: `This app allows to automatically provision subnets and IP addresses...`

We should add that these subnets and ip addresses will then be available as system defined variables (link the section on system defined variables in the readme) and that the purpose of this app is to allow users to automatically provision and configure specific subnets and ip addresses to the devices without the need of manual intervention.

Reply:
I think users will not be interested in `system defined varaibles` in this section of readme. It will make more sense to it in  **how-to-configure-automatic-provisioning-of-subnets-and-ips** section.

Reply:
ok, but I still suggest to quickly mention that these subnets and IP addresses will be available as configuration variables that can be used in templates, because that provides a better overview of what the features provided and users should be able to understand just by reading this statement, then they can drill down.